app/models/reviewable_user.rb

```diff
@@ -36,6 +36,38 @@ def perform_approve_user(performed_by, args)
     create_result(:success, :approved)
   end
 
+  def scrub(reason, guardian)
+    scrubbed_at = Time.zone.now
+    # We need to scrub the UserHistory record for when this user was deleted, as well as this reviewable's payload
+    UserHistory
+      .where(action: UserHistory.actions[:delete_user])
+      .where("details LIKE :query", query: "%\nusername: #{payload["username"]}\n%")
+      .where(created_at: (updated_at - 10.minutes)..(updated_at + 10.minutes))
+      .update_all(
+        details:
+          I18n.t(
+            "user.destroy_reasons.reviewable_details_scrubbed",
+            staff: guardian.current_user.username,
+            reason: reason,
+            timestamp: scrubbed_at,
+          ),
```

Comment:
This will cause the details to be saved in the admin's selected locale which may not be ideal. What we do for notifications is to store the details required for the translation in the record and generate the translation based on the locale of the user viewing the history.

Reply:
I wasn't sure about that: the `details` field seems to be used as a free-form text field: strings are generally inserted translated. There are quite a few examples across the codebase, the `StaffActionLogger` calls in `UsersController#activate` and `#deactivate` are a typical usage.

Reply:
I opened up an internal discussion but I realised that the `reviewable_user` payload stores all the information we need to regenerate the translation so we still have a way forward in the future.